precompiles/assets-erc20/src/lib.rs

```diff
@@ -491,74 +370,65 @@ where
 		.record(handle)?;
 
 		// Build output.
-		Ok(succeed(EvmDataWriter::new().write(true).build()))
+		Ok(true)
 	}
 
+	#[precompile::public("name()")]
 	fn name(
 		asset_id: AssetIdOf<Runtime, Instance>,
 		handle: &mut impl PrecompileHandle,
-	) -> EvmResult<PrecompileOutput> {
+	) -> EvmResult<UnboundedBytes> {
```

Comment:
Is this possible'
```suggestion
	) -> EvmResult<BoundedBytes<<Runtime as pallet_assets::Config>::StringLimit::get()> {
```

Reply:
The bound is only ensured when parsing calldata.